platform/darwin/scripts/style-spec-overrides-v8.json

```diff
@@ -1,4 +1,5 @@
 {
+  "light-cocoa-doc": "An `MGLLight` object represents the light source for extruded geometries in `MGLStyle`.",
```

Comment:
This is fine, but note that there’s no need to put every piece of Darwin-specific documentation in this file. The important thing is to put overrides here – things that would otherwise automatically say something different if we removed the entry from this file.